src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -606,7 +606,7 @@ class NodeSelectorKeyValue(schema.Base):
 
 class KubernetesCredentials(schema.Base):
     host: str
-    cluster_ca_certifiate: str
+    cluster_ca_certificate: str
```

Comment:
We should catch this in the update step and update the spelling for users if they had this set explicitly.

Reply:
I would add a `# typos: ignore` here for now, and a todo to address this for next release, so that we add as part of the update method as suggested. to keep this in mind, we could pin the issue in our notes, or in the issues tab